src/compute.ts

```diff
@@ -108,7 +108,7 @@ export async function compute(governances: string[]) {
   const release = await mutex.acquire();
 
   try {
-    register.setCurrentBlock(register.getCurrentBlock() + 1n);
+    await currentBlockTracker.increaseCurrentBlock();
```

Comment:
Haven't tested it yet, but overall it looks good to me.

We can follow your suggestion (and probably should as otherwise we will hit database on every query to bump current block) and only do this if we are not ignoring compute request due to delay.

Reply:
I moved it into the for loop, i hope it is fine if we increase the block for each governance